src/common/constants.js

```diff
@@ -4,6 +4,29 @@ import HealthIcon from 'icons/HealthIcon';
 import HealthBanner from 'assets/user-banner-health.png';
 import ResearchBanner from 'assets/user-banner-research.png';
 import CommunityBanner from 'assets/user-banner-community.png';
+import { without } from 'lodash';
+
+const DEFAULT_PROFILE_FIELDS = [
```

Comment:
I'm not sure this is actually necessary here. It may be misleading as it's disconnected from the actual form rendering (it only has an impact on percentage filled). The UI does an exclusion of `jobTitle` & `institution` in the jsx, I'd be more apt to follow that lead and just do the same exclusion in the `setUser` side effect